project-icp/bee-pollinator-app
# Add more crop groups

### Overview
Updates the source CDL csv to allow for more distinct groups, resulting in a larger number of crops in the CDL. This is done to accommodate specific values for hive density for crops that would otherwise be grouped together (cherries vs orchard).  Note we only have density numbers for a few crop types. Some new crop types are added, which are derived from a set of current crop types.  They are the original + a cover crop, these have different HN/HF values from their base.

Connects #118 
Connects #121 

#### Source for types and density
`Spreadsheet can be found on the fileshare`

![screenshot from 2017-01-30 20 26 12](https://cloud.githubusercontent.com/assets/1014341/22448893/6300cac8-e72a-11e6-8656-e4fd41707c11.png)
![screenshot from 2017-01-30 20 25 49](https://cloud.githubusercontent.com/assets/1014341/22448894/630c57b2-e72a-11e6-84a9-305069e565e2.png)



### Testing
* Test creating new grouped csv `python  group_cdl_data.py`
  * New crop types + cover crop types should be added
  * Test that the output is the version that is included in this PR
  * Check that the mapping json also updated to reflect the changes described
* Reclassify the raster based on the new grouped csv
  * Check that the distinct types have been preserved in the new raster
    * For example, Blueberry (`57`) is it's own type, and is predominant around Hammonton, NJ
* Recolor the reclassified raster
  * Since these colors have lost some meaning, it is sufficient to check that the script still runs correctly
